service/history/workflow_task_handler_callbacks.go

```diff
@@ -79,7 +79,6 @@ type (
 			*historyservice.RecordWorkflowTaskStartedRequest) (*historyservice.RecordWorkflowTaskStartedResponse, error)
 		handleWorkflowTaskCompleted(context.Context,
 			*historyservice.RespondWorkflowTaskCompletedRequest) (*historyservice.RespondWorkflowTaskCompletedResponse, error)
-		verifyFirstWorkflowTaskScheduled(context.Context, *historyservice.VerifyFirstWorkflowTaskScheduledRequest) error
 		// TODO also include the handle of workflow task timeout here
```

Comment:
looks like we can remove the TODO as well :)

Reply:
I will when I remove the last one next :)